website_membership_random_order/templates/website_membership_templates.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="index" inherit_id="website_membership.index">
+        <xpath expr="//t[@t-if='not memberships_partner_ids']" position="after">
+            <t t-else="">
+                <div
+                    class="s_alert alert-delta clearfix w-100 s_alert_md"
+                    data-name="Alert"
+                >
+                    <i
+                        class="fa fa-2x fa-info-circle s_alert_icon"
+                        data-original-title=""
+                        title=""
+                        aria-describedby="tooltip715756"
+                    />
+                    <div class="s_alert_content">
+                        <p
+                            style="text-align: center"
+                        >This order of appearance is random and changes each time you refresh.
```

Comment:
This order of appearance is random and may change each time

Reply:
remove extra \n

Reply:
The extra `<br>` is on purpose, and the redaction is that way because there's no doubt that each time it varies.